### PR TITLE
chore: change Drum Machine based on @moT01 review

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

hey @mot01 :eyes:
 the table looks like this
 
image

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
LGTM 🎉

File: curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md [hunk 1]
```diff
@@ -10,55 +10,57 @@ demoType: onClick
...
 **Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
 
-1. You should have a `div` element with an id of `drum-machine` that contains all other elements.
+1. You should have a `div` element with an  `id`  of `drum-machine` that contains all other elements.
 
-2. Inside the `#drum-machine` element you should have another `div` with an id of `pad-bank`.
+2. Inside the `#drum-machine` element you should have another `div` with an `id` of `pad-bank`.
 
-3. Inside the `#drum-machine` element you should have a `p` element with an id of `display`.
+3. In `id` the `#drum-machine` element you should have a `p` element with an `id` of `display`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md:17]:
```suggestion
3. Inside the `#drum-machine` element you should have a `p` element with an `id` of `display`.
```

File: curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md [hunk 2]
```diff
@@ -10,55 +10,57 @@ demoType: onClick
...
+5. Each `.drum-pad` should have an `audio` element which has a class of `clip`, a `src` attribute that points to an audio clip, and an `id` corresponding to the inner text of its parent `.drum-pad` element (e.g. `id="Q"`, `id="W"`, `id="E"` etc.).
 
 6. When you click on a `.drum-pad` element, the audio clip contained in its child audio element should be triggered.
 
 7. When you press the trigger key associated with each `.drum-pad`, the audio clip contained in its child audio element should be triggered (e.g. pressing the `Q` key should trigger the drum pad which contains the string `Q`, pressing the `W` key should trigger the drum pad which contains the string `W`, etc.).
 
 8. When a `.drum-pad` is triggered, you should display a string describing the associated audio clip as the inner text of the `#display` element (each string must be unique).
 
-Here are some audio samples you can use for your drum machine:
+Some audio samples you can use for your drum machine can be found at `https://cdn.freecodecamp.org/curriculum/drum/fileName`, where `fileName` is as follow:
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md:29]:
```suggestion
Some audio samples you can use for your drum machine can be found at `https://cdn.freecodecamp.org/curriculum/drum/<fileName>`, where `<fileName>` is as follows:
```

File: curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md [hunk 3]
```diff
@@ -10,55 +10,57 @@ demoType: onClick
...
 
-You should have a `div` element with an id of `drum-machine` that contains all other elements.
+You should have a `div` element with an `id` of `drum-machine` that contains all other elements.
 
 ```js
 assert.isNotNull(document.querySelector('div#drum-machine'));
 ```
 
-Inside the `#drum-machine` element you should have another `div` with an id of `pad-bank`.
+In `id` the `#drum-machine` element you should have another `div` with an `id` of `pad-bank`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md:51]:
```suggestion
Inside the `#drum-machine` element you should have another `div` with an `id` of `pad-bank`.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by fhsinchy — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md [hunk 4]
```diff
@@ -10,55 +10,57 @@ demoType: onClick
 
 **Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
 
-1. You should have a `div` element with an id of `drum-machine` that contains all other elements.
+1. You should have a `div` element with an  `id`  of `drum-machine` that contains all other elements.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/lab-drum-machine/6762ec275cef87635acc4fe3.md:13]:
```suggestion
1. You should have a `div` element with an `id` of `drum-machine` that contains all other elements.
```

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**